src/FluentNHibernate/FluentNHibernate.csproj

```diff
@@ -13,22 +13,26 @@
     <Description>FluentNHibernate</Description>
   </PropertyGroup>
   
-  <Import Project="$(MSBuildThisFileDirectory)..\Shared.msbuild" />
   
   <PropertyGroup>
     <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
-  
-  <ItemGroup>
-    <PackageReference Include="NHibernate" Version="5.0.3" />    
-  </ItemGroup>
-  
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net461' ">
+
+  <Import Project="$(MSBuildThisFileDirectory)..\Shared.msbuild" />
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
+    <PackageReference Include="NHibernate" Version="5.0.3" />

```

Comment:
I think this should be 5.1.0 as well now? Because all the tests are based on 5.1.0 so we should target this entire release to that? 5.1.0 does support net461 as well